src/app/pages/my-request-detail/my-request-detail.component.html

```diff
@@ -0,0 +1,74 @@
+
+<!--
+Copyright 2022 University of Oxford
+and Health and Social Care Information Centre, also known as NHS Digital
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+SPDX-License-Identifier: Apache-2.0
+-->
+<div *ngIf="request">
+  <div class="main-row hero">
+    <h1>{{ request.label }}</h1>
+  </div>
+  <div class="container hero-top-spacer">
+    <div *ngIf="state === 'loading'" class="mdm-my-request__progress">
+      <mat-progress-bar color="primary" mode="indeterminate"></mat-progress-bar>
```

Comment:
I don't think I've seen a progress bar (or any progress indicator) when the page is loading. Is this correct? Because it would be good to have one.